tests/InputFiles/PrintMinimumGridSpacing/Input3D.yaml

```diff
@@ -4,6 +4,9 @@
 # Executable: PrintMinimumGridSpacing
 # Check: execute
 
+Dimension:
+  Value: 3
+
```

Comment:
Update: Instead of specifying the number of dimensions as a command line argument, I created a custom tag to the source file so that it can be parsed from an input file, like above. When I run the executable with the these test yaml files, they run properly. However, the executable fails the 3 parse tests when `ctest` is run for Input1D, Input2D, and Input3D. Example verbose output from the Input1D.yaml parse test failing:

```terminate called after throwing an instance of 'boost::exception_detail::clone_impl<boost::exception_detail::error_info_injector<boost::program_options::unknown_option> >'
what():  unrecognised option '--check-options'
Test  #27: InputFiles.PrintMinimumGridSpacing.Input1D.yaml.parse ......................Child aborted***Exception```